rosidl_typesupport_freertps_cpp/bin/rosidl_typesupport_freertps_cpp

```diff
@@ -27,29 +25,10 @@ def main(argv=sys.argv[1:]):
         '--generator-arguments-file',
         required=True,
         help='The location of the file containing the generator arguments')
-    #parser.add_argument(
-    #    '--dds-interface-base-path',
-    #    required=True,
-    #    help='The base location of the DDS interface files')
     args = parser.parse_args(argv)
 
     generator_args = read_generator_arguments(args.generator_arguments_file)
-
-    try:
-        rc = generate_typesupport_freertps_cpp(generator_args)
-    except UnknownMessageType as e:
-        print(str(e), file=sys.stderr)
-        return 1
-    if rc:
-        return rc
-    return generate_dds_freertps_cpp(
-        generator_args['package_name'],
-        generator_args.get('additional_files', []),
-        #args.dds_interface_base_path,
-        generator_args.get('ros_interface_dependencies', []),
-        generator_args['output_dir']
-    )
-
+    return generate_typesupport_freertps_cpp(generator_args)
```

Comment:
We could still change the pattern used in this package if you think it's better, and ticket the change to be made in the other generators. Why do you think the other approach was chosen for the other generators?